services/collaboration/README.md

```diff
@@ -0,0 +1,25 @@
+# Collaboration
+
+The collaboration service connects ocis with document servers such as collabora and onlyoffice using the WOPI protocol.
+
+Since this service requires an external service (onlyoffice, for example), it won't run by default with the general `ocis server` command. You need to run it manually with the `ocis collaboration server` command.
+
+## Requirements
+
+The collaboration service requires the target document server (onlyoffice, collabora, etc) to be up and running.
+Some oCIS services are also required to be running in order to register the GRPC service for the "open in app" action:
+* reva's gateway
+* app provider
+
+If any of those services are down, the collaboration service won't start.
+For convenience, you might want to start the whole oCIS stack before starting the collaboration service.
```

Comment:
This is an interresting sentence as it highlights the possibility that the collaboration service is NOT part of the services started up automatically in the binary build. I may be wrong...

Reply:
I think it's commented above, in the second paragraph of the readme. We'll need to connect to an external server (onlyoffice, collabora, etc) which might not be available, and it will require configuration (we can't provide a valid default), that's why you need to start the service manually.